src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFCommand.groovy

```diff
@@ -233,7 +217,7 @@ class LSFCommand extends Command {
     private String prepareParentJobs(List<BEJobID> jobIds) {
         List<BEJobID> validJobIds = BEJob.uniqueValidJobIDs(jobIds)
         if (validJobIds.size() > 0) {
-            String joinedParentJobs = validJobIds.collect { "done(${it})" }.join(" && ")
+            String joinedParentJobs = validJobIds.collect { "exit(${it}, 0)" }.join(" && ")
```

Comment:
What I initially intended was, that you can control job dependencies using processingcommands. Never really was used but we might want to support this mechanism in some other way in the future.

Reply:
But what are ProcessingCommands?

Reply:
As said above, they were used to control the submission behaviour of Roddy jobs. But as also mentionend above, the name never was good. So SubmissionDirectives would be much more descriptive.